README.md

```diff
@@ -1,58 +1,71 @@
-<span style="font-size:larger;">DeePMD-kit Manual</span>
-========
+# DeePMD-kit Manual
+
 [![GitHub release](https://img.shields.io/github/release/deepmodeling/deepmd-kit.svg?maxAge=86400)](https://github.com/deepmodeling/deepmd-kit/releases)
 [![Documentation Status](https://readthedocs.org/projects/deepmd/badge/?version=latest)](https://deepmd.readthedocs.io/en/latest/?badge=latest)
 
+> [!Caution]
+> ⚠️⚠️⚠️
+>
+> **The current branch is no longer maintained due to the outdated Paddle version it adapts to and the significant differences from the DeePMD-kit main branch.**
+>
+> **The new paddle backend branch is [DeePMD-kit(paddle branch)](https://github.com/deepmodeling/deepmd-kit/tree/paddle)**
```

Comment:
Shouldn't the branch be `paddle2`?

Reply:
> Shouldn't the branch be `paddle2`?

I guess paddle2 is not suitable name in the future, and code in old paddle can be removed or moved to other branch name, so can we add extra deprecation note, such as: "As descripbed above, the current `paddle` branch will be replaced with the content in `paddle2` branch after 2024.x.y". and replace code in `paddle` branch with code in `padddle2 then delete `paddle2` branch on 2024.x.y that day?

Reply:
I think it is not necessary if we want to rename the branch

Reply:
> I think it is not necessary if we want to rename the branch

URL has been updated

Reply:
> I think it is not necessary if we want to rename the branch

paddle branch will not be maintain any more and the code in paddle branch is too old to run. so I guess it is necessary to remove paddle branch after certain day and move paddle2's code into paddle branch

Reply:
I will rename it tomorrow. I prefer keeping the old branch for the recording purposes. Even if no one uses it, people can know the history of development through the  old branch.

#By the way, we will focus on supporting multiple backends in the next five weeks. After we refactor several codes, it's more easier to support Paddle in the main development branch.

Reply:
> I will rename it tomorrow. I prefer keeping the old branch for the recording purposes. Even if no one uses it, people can know the history of development through the old branch.
> 
> By the way, we will focus on supporting multiple backends in the next five weeks. After we refactor several codes, it's more easier to support Paddle in the main development branch.

1. I agree with you much that we need to record old code and history of it even if no one use(so there indeed unnecessary to delete old code), but I guess it is important to move the old code to some "bakcup" branch but not be kept in offcial branch(`paddle` branch), since an official branch is important or will cause recognition error, and we always prefer showing newest runnable code for those who use DeePMD-kit with paddle backend in official branch.

2. It is very glad to hear that a multi-backend project is released and you are working for it, willing to help to build togather after multi-bakend is finished.

Thanks for your reply.

Reply:
I've renamed `paddle` to `paddle-backup` and `paddle2` to `paddle`.